Ports/build_installed.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
+source .hosted_defs.sh
```

Comment:
I think we use the `.` syntax to source scripts.

Reply:
Alright